cpp/src/rolling/grouped_rolling.cu

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
+#include <rolling/rolling_detail.cuh>
+#include <rolling/rolling_jit_detail.hpp>
```

Comment:
Include files in the same directory as the current file using "local_filename.hpp", not angle brackets. The path `<rolling/rolling_jit_detail.hpp>` assumes that `/cudf/src` is specified to the compilation `-I`. But the path `"rolling_jit_detail.hpp"` does not make that assumption, and looks in the current file's directory.

```suggestion
#include "rolling_detail.cuh"
#include "rolling_jit_detail.hpp"
```